### PR TITLE
added notes about unit tests variables

### PR DESCRIPTION
**Pull request by mauno-ronkko**
notes about where to introduce and initialize unit test variables

### CODE REVIEW

File: codeconventions.md [hunk 1]
```diff
@@ -42,6 +42,10 @@ ODA source code is developed on various platforms and editors. Therefore common
 - arrange test code to follow http://wiki.c2.com/?ArrangeActAssert pattern
   + NOTE: always add empty line before and after Act-section even if test does not contain Arrange or Assert section(s) (e.g. if test is expected to throw it does not need Assertions) to make clear separation with Act-section and missing sections.
   + this may not be feasible in more complex integration tests where empty lines may be needed inside sections to make tests more readable.
+- Regarding variables in tests:
+  1. If a variable is used in (almost) all tests, introduce it as an initialized final object attribute.
```
Comment (ruoat) [inline codeconventions.md]: object attribute -> field tai member variable
